vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -37,6 +37,8 @@ samba_domain: "DC.ARKCASE-CE"
 # server_admin goes in the httpd conf file, and is the email
 # address of the arkcase-admin user
 server_admin: "david.miller@arkcase.com"
+# Not using arkcase-admin as a hardcoded user but instead use it as a variable since this user can vary from environment to environment
+admin_user: "arkcase-admin"
```

Comment:
Why don't you use the existing **arkcase_admin_user** variable? Now we have to add this new variable in all templates. Can we use arkcase_admin_user as a default?

Reply:
I agree, we already have the arkcase_admin_user: "{{ ldap_user_prefix }}arkcase-admin"